project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.3.5
+sbt.version = 1.2.8
```

Comment:
This would not compile for me locally due to a coursier exception, until I downgraded the sbt version.